src/backend/index.ts

```diff
@@ -11,6 +11,7 @@ import changeRequestsRouter from './src/routes/change-requests.routes';
 import descriptionBulletsRouter from './src/routes/description-bullets.routes';
 import tasksRouter from './src/routes/tasks.routes';
 import reimbursementRequestsRouter from './src/routes/reimbursement-requests.routes';
+import designReviewRouter from './src/routes/design-reviews.routes';
```

Comment:
Why are all the file names now plural? On develop they're still singular, which is going to cause mad merge conflicts. If they're supposed to be plural then you should also make sure the class names match (e.x. designReviewRouter -> designReviewsRouter)